### DISCUSSION TITLE
How should I delete the IP whitelist in the client ui interface, can only be modified through the source code

### DISCUSSION THREAD

**Discussion by rrc11111** [rustdesk/rustdesk#6670] [Q&A]
How should I delete the IP whitelist in the client ui interface, can only be modified through the source code

**Comment by yaroz** [comment]
You click on the "Use IP whitelisting" text, and it pops open a window.  It'd be nice if they added a button to modify it, if the box is checked... or added a multi-line box under.

**Reply by rrc11111** [reply]
I set up an ip whitelist, but my access was denied
